src/extra/AutoUi/schemaOps.ts

```diff
@@ -43,6 +43,7 @@ export interface AutoUIAction<T> {
 		affectedEntries?: T[];
 		onDone: (isSuccessful: boolean) => void;
 	}) => React.ReactNode;
+	actionFn?: (props: { affectedEntries?: T[] }) => void;
```

Comment:
I like it! I would just also make the `renderer` function to not be required, so we can avoid having renderer: null, actionFn: () => ...